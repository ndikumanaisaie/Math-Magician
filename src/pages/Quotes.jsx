import React from 'react';

const Quotes = () => (
  <div>Quotes</div>
);

export default Quotes;
