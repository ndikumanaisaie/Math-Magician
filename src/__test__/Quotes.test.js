import React from 'react';
import renderer from 'react-test-renderer';

import Quotes from '../pages/Quotes';

it('renders correctly the Quotes component', () => {
  const quotes = renderer.create(<Quotes />).toJSON();
  expect(quotes).toMatchSnapshot();
});
