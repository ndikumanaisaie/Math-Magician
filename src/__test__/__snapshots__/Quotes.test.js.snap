// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the Quotes component 1`] = `
<div
  className="quote"
>
  <p>
    Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. â€” Richard Courant, German-American mathematician
  </p>
</div>
`;
